client/styles/components/footer.css

```diff
@@ -0,0 +1,15 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+@tailwind variants;
+
+.footer{
+    @apply bg-neutral-50 border-t border-neutral-200
+}
+
+.footer-container{
```

Comment:
it's doing more then one thing

it should be juts footer, or container.

Reply:
i wanted to be refereed about footer's container not be a footer and also a container